sip/sip-001-burn-election.md

```diff
@@ -17,25 +17,18 @@ License: BSD 2-Clause
 ## Abstract
 
 This proposal describes a mechanism for single-leader election using
-_tunable proofs_.  Tunable proofs are a mechanism for bootstrapping a new
-blockchain on top of an existing blockchain by (1) destroying the existing
-blockchain's tokens (generating a _proof-of-burn_) and (2) generating a separate
-accompanying proof-of-work.  The tunable proof is so named because it is derived
-from a weighted sum of these two proofs.
+_proof-of-burn_ (PoB).  Proof of burn is a mechanism for bootstrapping a new
+blockchain on top of an existing blockchain by rendering the tokens unspendable
+(i.e. "burning" them).
 
-Tunable proofs are concerned with deciding which Stacks block miner (called _leader_ in this text) is elected for
+Proof of burn is concerned with deciding which Stacks block miner (called _leader_ in this text) is elected for
 producing the next block, as well as deciding how to resolve
-conflicting transaction histories.  The protocol assigns a tunable proof an
-aggregate score from its proofs-of-burn and proofs-of-work, which is used to
+conflicting transaction histories.  The protocol assigns a score to each leader
+based on the fraction of tokens it burned, which is used to
 (1) probabilistically select the next leader proportional to its normalized score
 and to (2) rank conflicting transaction histories by their aggregate scores in order
```

Comment:
> (2) rank conflicting transaction histories by their aggregate scores in order

This has also changed --- we rank conflicting histories by their total number of epochs.

Reply:
Thanks; fixed